Session.php

```diff
@@ -187,7 +192,7 @@ public function destroySession($id)
     public function gcSession($maxLifetime)
     {
         $this->db->getCollection($this->sessionCollection)
-            ->remove(['expire' => ['$lt' => time()]]);
+            ->remove(['expire' => ['$lt' => new \MongoDate()]]);
```

Comment:
@klimov-paul, All right. When using the index, the garbage collector doesn't needed.